packages/checkbox/src/styles.tsx

```diff
@@ -0,0 +1,160 @@
+import * as tinycolor from "tinycolor2"
+import {DefaultTheme} from "styled-components/macro"
+import {getLinearGradientWithStates} from "./helpers"
+
+export type AppearanceType = "primary" | "minimal" | "card"
+
+export const getLabelStyle = (
+  appearance: AppearanceType,
+  disabled: boolean = false,
+  checked: boolean = false
+) => (_: {theme: DefaultTheme}) => {
+  const {colors, fills, scales} = _.theme
+  const disabledAppearance = {
+    color: colors.text.muted
+  }
+  const disabledCardAppearance = {
```

Comment:
Disabled checkbox should also have a border but transparent. Try to change story between disabled and not - label jumps 2px to right